components/table-filter/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/pingcap/tidb-tools/components/table-filter
+
+require (
+	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
+	github.com/pingcap/errors v0.11.5-0.20200917111840-a15ef68f753d
+)
+
+go 1.14
```

Comment:
we use 1.13 other place, should we keep same go version?

Reply:
maybe.